### PR TITLE
CapFrameX: fix autoupdate extract_dir

### PR DESCRIPTION
**Pull request by pratikpc**
@404NetworkError could you please help us out in fixing CapFrameX?

Is this fix correct?

Is the application 32 bit or 64 bit?

### CODE REVIEW

**Comment by 404NetworkError** [comment]
On my 64-bit machine, it appears to be 64-bit.

**Comment by pratikpc** [comment]
Their README also mentions Solution Platform x64

Will make the change!

Thanks

**Comment by pratikpc** [comment]
@404NetworkError could you check it offline once?

Also could you check if the existing CapFrameX push works?

File: bucket/capframex.json [hunk 1]
```diff
@@ -6,9 +6,13 @@
...
         "OCAT": "ocat"
     },
-    "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.6.6/CapFrameX_v1.6.6_Portable.zip",
-    "hash": "f889bd33d52c05172f79d18579cc2249b3e1c750a6ec4daef735e840f637608b",
-    "extract_dir": "CapFrameX 1.6.6 Portable",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.6.6/CapFrameX_v1.6.6_Portable.zip",
+            "hash": "f889bd33d52c05172f79d18579cc2249b3e1c750a6ec4daef735e840f637608b",
+            "extract_dir": "CapframeX 1.6.6 Portable"
```
Comment (404NetworkError) [inline bucket/capframex.json:13]: Needs to be `CapFrameX 1.6.6 Portable`; missed the capital `F`.

**Comment by 404NetworkError** [comment]
Everything else works.

File: bucket/capframex.json (hunk 1 above)
Comment (pratikpc) [reply bucket/capframex.json:13]: Thanks!!!!

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**